Uchu.StandardScripts/NinjagoMonastery/GoToNpcBodge.cs

```diff
@@ -0,0 +1,51 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.NinjagoMonastery
+{
+    [ZoneSpecific(2000)]
+    public class GoToNpcBodge : NativeScript
+    {
+        public override Task LoadAsync()
+        {
+            // Okay look this is a really stupid script but the cdclient is messed up.
+            // Check the lu-explorer links I left here. The GoToNpc tasks have
+            // entirely unrelated targets. (These targets aren't used for any other
+            // GoToNpc tasks, so it should be fine to just call GoToNpc for them.)
+
+            // https://lu.lcdruniverse.org/explorer/missions/1797
+            const int senseiWuLot = 13789;
```

Comment:
I know for sure that there's a LOT enum, for readability might be nice to store these there :)

Reply:
Done, thanks!